log/logger_multiwriter.go

```diff
@@ -58,6 +65,7 @@ func (mw *multiWriter) Write(p []byte) (n int, err error) {
 	}
 
 	for range mw.writers {
+		// NOTE: These results do not necessarily reflect the current io.writer
```

Comment:
What do you mean?

Reply:
Due to the go scheduler and writers finishing at different times the response coming from the channel might not match up with the for loop writer, even though it's not an issue. It was mainly for me, if I wanted to wrap more details.

Reply:
Nice! Sounds like it would be good to add your explanation 🎉 
```suggestion
		// NOTE: These results do not necessarily reflect the current io.writer due to the go scheduler and writers 
		// finishing at different times, the response coming from the channel might not match up with the for loop writer
```